hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HddsServerUtil.java

```diff
@@ -451,6 +452,7 @@ public static String getDatanodeIdFilePath(ConfigurationSource conf) {
    */
   public static SCMSecurityProtocolClientSideTranslatorPB getScmSecurityClient(
       ConfigurationSource conf) throws IOException {
+    new SecurityConfig(conf);
```

Comment:
Are these needed for the same reason as here? https://github.com/apache/ozone/blob/217f37bca42403f38847bfdd4bd110083949dc0c/hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneSecurityUtil.java#L65-L67

Reply:
Yes, unfortunately yes.

I am not the fan of this scattered solution either, but different initialization paths may or may not create a SecurityConfig object, this way we ensure that the BouncyCastle JCE provider is loaded programmatically, so we can use it to convert between PEM files and Certificates.